demonstrations/ahs_aquila.py

```diff
@@ -0,0 +1,1365 @@
+specifications and where they are relevant as we go through this demo.
+
+.. note:: Some cells of this notebook will only run when hardware is online. If you want to run it at
+other times to experiment with the concepts, the hardware device can be switched out with the Braket
+simulator. When interpreting the section of the demo regarding discretization for hardware, bear in
+mind that the simulator does not discretize the functions before upload, and so will not accurately
+demonstrate the discretization behaviour.
+
+But before we get into any details, lets take a moment to get familiar with the physical system we
+will be interacting with, and specifically the Hamiltonian describing it. In this treatment of the
+system Hamiltonian for Rydberg atoms, we will assume that we are operating such that we only allow
+access to two states; the low and high energy states are referred to as the ground and Rydberg states
+respectively.
+
+Constructing a pulse program to run on the Aquila hardware is done in two steps, each of which allows us to modify
+different parts of the system Hamiltonian:
+
+1. Define atom positions, which determines qubit connectivity
+2. Specify the quantum evolution via the drive parameters
+
+Let's start with the atom positions and the resulting Hamiltonian term describing inter-qubit interactions.
+
+
+Interaction term and atom arrangement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Qubit interaction in a Rydberg atom system is mediated by a mechanism called Rydberg blockade, which arises
+due to van der Waals forces between the atoms. In the Aquila system, to modify these interactions, we define
+a **register** (the layout of the atoms) by providing coordinates.
+
+If the atoms are placed at a significant distance from one another, they operate as essentially independent
+systems, and we can easily drive all of them to the Rydberg state simultaneously. However, as we decrease the
+distance between atoms, it becomes harder and harder to excite neighboring atoms.
+
+Below is a conceptual diagram demonstrating this interaction for a pair of atoms. At a distance, a similar
+energetic cost is paid to from 0 to 1 excitation and from 1 to 2 excitations. However, as we move the
+atoms into closer proximity, we see a rapidly increasing energy cost to drive to the doubly excited state.
+
+|
+
+.. figure:: ../demonstrations/ahs_aquila/rydberg_blockade_diagram.png
+    :align: center
+    :scale: 30%
+    :alt: A diagram of the energy levels for the ground, single excitation, and double excitation states
+    :target: javascript:void(0);
+
+|
+
+Mathematically, this interaction is described by the following Hamiltonian:
+
+.. math:: H_{vdW, j, k} = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} V_{jk}n_jn_k = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} \frac{C_6}{R^6_{jk}}n_jn_k
+
+where :math:`R_{jk} = \lvert x_j - x_k \lvert` is the distance in meters between atoms *j* and *k*, and :math:`C_6`
```

Comment:
```suggestion
where :math:`R_{jk} = \lvert x_j - x_k \lvert` is the distance in metres between atoms *j* and *k*, and :math:`C_6`
```
(I think this is the standard Canadian spelling)

Reply:
That looks so weird to me but Google agrees with you so I accept